src/celengine/astro.cpp

```diff
@@ -321,15 +373,19 @@ astro::Date::Date(double jd)
     tzname = "UTC";
 }
 
-const char* astro::Date::toCStr(Format format) const
+
+const char*
+Date::toCStr(Format format) const
 {
-    static char date[255];
+    static std::array<char, 255> date;
```

Comment:
if you're in mood could you make this thread safe? I hope one day we can start using more then 1 core.

Reply:
Yep, makes sense. Usages don't seem to be on any particularly critical paths, so might as well just return a `std::string`